__tests__/app.test.js

```diff
@@ -458,3 +458,38 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/users", () => {
+  it("GET 200: repsonds with array of objects of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const result = body.users;
+        expect(result).toBeInstanceOf(Array);
+        result.forEach((user) => {
```

Comment:
always make sure to check the length of the the array you get back before iterating over it or else you may get a false pass.